### DISCUSSION TITLE
Additional typing required for using schema.org?

### DISCUSSION THREAD

**Discussion by briesenberg07** [uwlib-cams/uwlswd#48] [modeling]
It occurs to me that we are using terms from schema.org but nowhere\* are UWLSWD resources typed as [schema:Thing](https://schema.org/Thing)s (or a subclass thereof--I imagine that we'd use [schema:Dataset](https://schema.org/Dataset)).

Is such typing needed to satisfy schema.org property requirements? 

It may not be, as these can be very permissive. See for example schema:disambiguatingDescription, which uses schema:domainIncludes in lieu of rdfs:domain:

```
schema:disambiguatingDescription a rdf:Property ;
    rdfs:label "disambiguatingDescription" ;
    rdfs:comment "A sub property of description. A short description of the item used to disambiguate from other, similar items. Information from other properties (in particular, name) may be necessary for the description to be useful for disambiguation." ;
    rdfs:subPropertyOf schema:description ;
    schema:domainIncludes schema:Thing ;
    schema:rangeIncludes schema:Text .

```

\* *To be confirmed with a query! But I feel confident in saying **at least** that UWLSWD resources are not **consistently** typed using schema.org classes*

**Comment by gerontakos** [comment]
Given schema.org formalizes permissiveness, and given that it is not an ontology (it's more like a "tag suite" to enhance search engines) we can do whatever we want! Right?

However, as you know, in semantic web, simply using a property with a domain entails a type for the thing being described. To say 
<someBook> <hasTitleOfWork> "some title" 
where the ontology states
<hasTitleOfWork> rdfs:domain <Work>
means <someBook> is a Work, no matter what else we say. It does not have to be explicit. 

As the schema.org types add nothing of value I can see -- especially schema:Thing (which is everything), and schema:Dataset just seems redunadant) -- I'd say leave it out.

**Comment by briesenberg07** [comment]
📢 We won't add the schema:Dataset type to our resources at this time
